### ISSUE TITLE
[ERROR] No documents exist - with latest docker image

### ISSUE THREAD

**Issue by githubhs17** [dependency-check/DependencyCheck#1735]
09:53:28 Status: Downloaded newer image for owasp/dependency-check:latest
[Pipeline] sh
09:53:29 + docker run .......
09:54:27 [INFO] Checking for updates
09:54:30 [INFO] NVD CVE requires several updates; this could take a couple of minutes.
09:54:30 [INFO] Download Started for NVD CVE - 2002
09:54:30 [INFO] Download Started for NVD CVE - 2007
09:54:30 [INFO] Download Started for NVD CVE - 2005
09:54:30 [INFO] Download Started for NVD CVE - 2004
09:54:30 [INFO] Download Started for NVD CVE - 2006
09:54:30 [INFO] Download Started for NVD CVE - 2003
09:54:30 [INFO] Download Started for NVD CVE - 2010
09:54:30 [INFO] Download Started for NVD CVE - 2009
09:54:30 [INFO] Download Started for NVD CVE - 2008
09:54:32 [INFO] Download Complete for NVD CVE - 2003  (1893 ms)
09:54:32 [INFO] Download Started for NVD CVE - 2011
09:54:32 [INFO] Processing Started for NVD CVE - 2003
09:54:32 [INFO] Download Complete for NVD CVE - 2005  (2332 ms)
09:54:33 [INFO] Download Started for NVD CVE - 2012
09:54:33 [INFO] Processing Started for NVD CVE - 2005
09:54:33 [INFO] Download Complete for NVD CVE - 2004  (2483 ms)
09:54:33 [INFO] Download Complete for NVD CVE - 2002  (2542 ms)
09:54:33 [INFO] Download Started for NVD CVE - 2013
09:54:33 [INFO] Processing Started for NVD CVE - 2002
09:54:33 [INFO] Download Started for NVD CVE - 2014
09:54:33 [INFO] Processing Started for NVD CVE - 2004
09:54:33 [INFO] Download Complete for NVD CVE - 2008  (2686 ms)
09:54:33 [INFO] Download Started for NVD CVE - 2015
09:54:33 [INFO] Download Complete for NVD CVE - 2006  (2762 ms)
09:54:33 [INFO] Processing Started for NVD CVE - 2008
09:54:33 [INFO] Download Started for NVD CVE - 2016
09:54:33 [INFO] Processing Started for NVD CVE - 2006
09:54:33 [INFO] Download Complete for NVD CVE - 2009  (2785 ms)
09:54:33 [INFO] Download Started for NVD CVE - 2017
09:54:33 [INFO] Download Complete for NVD CVE - 2007  (2869 ms)
09:54:33 [INFO] Download Started for NVD CVE - 2018
09:54:33 [INFO] Download Complete for NVD CVE - 2010  (3212 ms)
09:54:33 [INFO] Download Started for NVD CVE - 2019
09:54:35 [INFO] Download Complete for NVD CVE - 2019  (1752 ms)
09:54:35 [INFO] Download Started for NVD CVE - Modified
09:54:36 [INFO] Download Complete for NVD CVE - 2015  (2824 ms)
09:54:36 [INFO] Download Complete for NVD CVE - 2011  (4355 ms)
09:54:36 [INFO] Download Complete for NVD CVE - 2017  (3682 ms)
09:54:37 [INFO] Download Complete for NVD CVE - 2018  (3961 ms)
09:54:37 [INFO] Download Complete for NVD CVE - Modified  (1847 ms)
09:54:44 [INFO] Download Complete for NVD CVE - 2013  (11134 ms)
09:54:44 [INFO] Processing Complete for NVD CVE - 2003  (12271 ms)
09:54:44 [INFO] Processing Started for NVD CVE - 2009
09:54:45 [INFO] Download Complete for NVD CVE - 2016  (11995 ms)
09:54:45 [INFO] Download Complete for NVD CVE - 2012  (12961 ms)
09:54:48 [INFO] Download Complete for NVD CVE - 2014  (14642 ms)
09:54:52 [INFO] Processing Complete for NVD CVE - 2004  (18320 ms)
09:54:52 [INFO] Processing Started for NVD CVE - 2007
09:55:06 [INFO] Processing Complete for NVD CVE - 2005  (31735 ms)
09:55:06 [INFO] Processing Started for NVD CVE - 2010
09:55:11 [INFO] Processing Complete for NVD CVE - 2002  (38305 ms)
09:55:11 [INFO] Processing Started for NVD CVE - 2019
09:55:16 [INFO] Processing Complete for NVD CVE - 2019  (4142 ms)
09:55:16 [INFO] Processing Started for NVD CVE - 2015
09:55:19 [INFO] Processing Complete for NVD CVE - 2009  (34145 ms)
09:55:19 [INFO] Processing Started for NVD CVE - 2011
09:55:19 [INFO] Processing Complete for NVD CVE - 2006  (45847 ms)
09:55:19 [INFO] Processing Started for NVD CVE - 2017
09:55:21 [INFO] Processing Complete for NVD CVE - 2008  (47504 ms)
09:55:21 [INFO] Processing Started for NVD CVE - 2018
09:55:44 [INFO] Processing Complete for NVD CVE - 2007  (50648 ms)
09:55:44 [INFO] Processing Started for NVD CVE - Modified
09:55:45 [INFO] Processing Complete for NVD CVE - Modified  (3251 ms)
09:55:45 [INFO] Processing Started for NVD CVE - 2013
09:55:56 [INFO] Processing Complete for NVD CVE - 2010  (50623 ms)
09:55:56 [INFO] Processing Started for NVD CVE - 2016
09:56:29 [INFO] Processing Complete for NVD CVE - 2011  (70927 ms)
09:56:29 [INFO] Processing Started for NVD CVE - 2012
09:56:38 [INFO] Processing Complete for NVD CVE - 2015  (82302 ms)
09:56:38 [INFO] Processing Started for NVD CVE - 2014
09:56:58 [INFO] Processing Complete for NVD CVE - 2013  (71228 ms)
09:57:14 [INFO] Processing Complete for NVD CVE - 2016  (78103 ms)
09:57:15 [INFO] Processing Complete for NVD CVE - 2017  (115949 ms)
09:57:15 [INFO] Processing Complete for NVD CVE - 2012  (45364 ms)
09:57:16 [INFO] Processing Complete for NVD CVE - 2018  (115549 ms)
09:57:18 [INFO] Processing Complete for NVD CVE - 2014  (40552 ms)
09:57:18 [INFO] Begin database maintenance.
09:57:42 [INFO] End database maintenance.
09:57:44 [INFO] Check for updates complete (197388 ms)
09:57:46 [ERROR] Unable to continue dependency-check analysis.
09:57:46 [ERROR] One or more fatal errors occurred
09:57:46 [ERROR] No documents exist

From Log file:
2019-02-18 08:54:26,788 org.owasp.dependencycheck.Engine:904
INFO  - Checking for updates
2019-02-18 08:54:30,416 org.owasp.dependencycheck.data.update.NvdCveUpdater:247
INFO  - NVD CVE requires several updates; this could take a couple of minutes.
2019-02-18 08:54:30,420 org.owasp.dependencycheck.data.update.nvd.DownloadTask:123
INFO  - Download Started for NVD CVE - 2002
2019-02-18 08:54:30,422 org.owasp.dependencycheck.data.update.nvd.DownloadTask:123
INFO  - Download Started for NVD CVE - 2007
2019-02-18 08:54:30,422 org.owasp.dependencycheck.data.update.nvd.DownloadTask:123
INFO  - Download Started for NVD CVE - 2005
2019-02-18 08:54:30,422 org.owasp.dependencycheck.data.update.nvd.DownloadTask:123
INFO  - Download Started for NVD CVE - 2004
2019-02-18 08:54:30,422 org.owasp.dependencycheck.data.update.nvd.DownloadTask:123
INFO  - Download Started for NVD CVE - 2006
2019-02-18 08:54:30,420 org.owasp.dependencycheck.data.update.nvd.DownloadTask:123
INFO  - Download Started for NVD CVE - 2003
2019-02-18 08:54:30,423 org.owasp.dependencycheck.data.update.nvd.DownloadTask:123
INFO  - Download Started for NVD CVE - 2010
2019-02-18 08:54:30,423 org.owasp.dependencycheck.data.update.nvd.DownloadTask:123
INFO  - Download Started for NVD CVE - 2009
2019-02-18 08:54:30,422 org.owasp.dependencycheck.data.update.nvd.DownloadTask:123
INFO  - Download Started for NVD CVE - 2008
2019-02-18 08:54:32,325 org.owasp.dependencycheck.data.update.nvd.DownloadTask:137
INFO  - Download Complete for NVD CVE - 2003  (1893 ms)
2019-02-18 08:54:32,328 org.owasp.dependencycheck.data.update.nvd.DownloadTask:123
INFO  - Download Started for NVD CVE - 2011
2019-02-18 08:54:32,328 org.owasp.dependencycheck.data.update.nvd.ProcessTask:149
INFO  - Processing Started for NVD CVE - 2003
2019-02-18 08:54:32,760 org.owasp.dependencycheck.data.update.nvd.DownloadTask:137
INFO  - Download Complete for NVD CVE - 2005  (2332 ms)
2019-02-18 08:54:32,764 org.owasp.dependencycheck.data.update.nvd.DownloadTask:123
INFO  - Download Started for NVD CVE - 2012
2019-02-18 08:54:32,772 org.owasp.dependencycheck.data.update.nvd.ProcessTask:149
INFO  - Processing Started for NVD CVE - 2005
2019-02-18 08:54:32,913 org.owasp.dependencycheck.data.update.nvd.DownloadTask:137
INFO  - Download Complete for NVD CVE - 2004  (2483 ms)
2019-02-18 08:54:32,966 org.owasp.dependencycheck.data.update.nvd.DownloadTask:137
INFO  - Download Complete for NVD CVE - 2002  (2542 ms)
2019-02-18 08:54:33,016 org.owasp.dependencycheck.data.update.nvd.DownloadTask:123
INFO  - Download Started for NVD CVE - 2013
2019-02-18 08:54:33,023 org.owasp.dependencycheck.data.update.nvd.ProcessTask:149
INFO  - Processing Started for NVD CVE - 2002
2019-02-18 08:54:33,026 org.owasp.dependencycheck.data.update.nvd.DownloadTask:123
INFO  - Download Started for NVD CVE - 2014
2019-02-18 08:54:33,044 org.owasp.dependencycheck.data.update.nvd.ProcessTask:149
INFO  - Processing Started for NVD CVE - 2004
2019-02-18 08:54:33,123 org.owasp.dependencycheck.data.update.nvd.DownloadTask:137
INFO  - Download Complete for NVD CVE - 2008  (2686 ms)
2019-02-18 08:54:33,182 org.owasp.dependencycheck.data.update.nvd.DownloadTask:123
INFO  - Download Started for NVD CVE - 2015
2019-02-18 08:54:33,192 org.owasp.dependencycheck.data.update.nvd.DownloadTask:137
INFO  - Download Complete for NVD CVE - 2006  (2762 ms)
2019-02-18 08:54:33,192 org.owasp.dependencycheck.data.update.nvd.ProcessTask:149
INFO  - Processing Started for NVD CVE - 2008
2019-02-18 08:54:33,202 org.owasp.dependencycheck.data.update.nvd.DownloadTask:123
INFO  - Download Started for NVD CVE - 2016
2019-02-18 08:54:33,204 org.owasp.dependencycheck.data.update.nvd.ProcessTask:149
INFO  - Processing Started for NVD CVE - 2006
2019-02-18 08:54:33,220 org.owasp.dependencycheck.data.update.nvd.DownloadTask:137
INFO  - Download Complete for NVD CVE - 2009  (2785 ms)
2019-02-18 08:54:33,221 org.owasp.dependencycheck.data.update.nvd.DownloadTask:123
INFO  - Download Started for NVD CVE - 2017
2019-02-18 08:54:33,297 org.owasp.dependencycheck.data.update.nvd.DownloadTask:137
INFO  - Download Complete for NVD CVE - 2007  (2869 ms)
2019-02-18 08:54:33,304 org.owasp.dependencycheck.data.update.nvd.DownloadTask:123
INFO  - Download Started for NVD CVE - 2018
2019-02-18 08:54:33,645 org.owasp.dependencycheck.data.update.nvd.DownloadTask:137
INFO  - Download Complete for NVD CVE - 2010  (3212 ms)
2019-02-18 08:54:33,676 org.owasp.dependencycheck.data.update.nvd.DownloadTask:123
INFO  - Download Started for NVD CVE - 2019
2019-02-18 08:54:35,428 org.owasp.dependencycheck.data.update.nvd.DownloadTask:137
INFO  - Download Complete for NVD CVE - 2019  (1752 ms)
2019-02-18 08:54:35,435 org.owasp.dependencycheck.data.update.nvd.DownloadTask:123
INFO  - Download Started for NVD CVE - Modified
2019-02-18 08:54:36,007 org.owasp.dependencycheck.data.update.nvd.DownloadTask:137
INFO  - Download Complete for NVD CVE - 2015  (2824 ms)
2019-02-18 08:54:36,683 org.owasp.dependencycheck.data.update.nvd.DownloadTask:137
INFO  - Download Complete for NVD CVE - 2011  (4355 ms)
2019-02-18 08:54:36,904 org.owasp.dependencycheck.data.update.nvd.DownloadTask:137
INFO  - Download Complete for NVD CVE - 2017  (3682 ms)
2019-02-18 08:54:37,266 org.owasp.dependencycheck.data.update.nvd.DownloadTask:137
INFO  - Download Complete for NVD CVE - 2018  (3961 ms)
2019-02-18 08:54:37,283 org.owasp.dependencycheck.data.update.nvd.DownloadTask:137
INFO  - Download Complete for NVD CVE - Modified  (1847 ms)
2019-02-18 08:54:44,151 org.owasp.dependencycheck.data.update.nvd.DownloadTask:137
INFO  - Download Complete for NVD CVE - 2013  (11134 ms)
2019-02-18 08:54:44,599 org.owasp.dependencycheck.data.update.nvd.ProcessTask:160
INFO  - Processing Complete for NVD CVE - 2003  (12271 ms)
2019-02-18 08:54:44,599 org.owasp.dependencycheck.data.update.nvd.ProcessTask:149
INFO  - Processing Started for NVD CVE - 2009
2019-02-18 08:54:45,197 org.owasp.dependencycheck.data.update.nvd.DownloadTask:137
INFO  - Download Complete for NVD CVE - 2016  (11995 ms)
2019-02-18 08:54:45,725 org.owasp.dependencycheck.data.update.nvd.DownloadTask:137
INFO  - Download Complete for NVD CVE - 2012  (12961 ms)
2019-02-18 08:54:47,669 org.owasp.dependencycheck.data.update.nvd.DownloadTask:137
INFO  - Download Complete for NVD CVE - 2014  (14642 ms)
2019-02-18 08:54:51,365 org.owasp.dependencycheck.data.update.nvd.ProcessTask:160
INFO  - Processing Complete for NVD CVE - 2004  (18320 ms)
2019-02-18 08:54:51,366 org.owasp.dependencycheck.data.update.nvd.ProcessTask:149
INFO  - Processing Started for NVD CVE - 2007
2019-02-18 08:55:04,508 org.owasp.dependencycheck.data.update.nvd.ProcessTask:160
INFO  - Processing Complete for NVD CVE - 2005  (31735 ms)
2019-02-18 08:55:04,509 org.owasp.dependencycheck.data.update.nvd.ProcessTask:149
INFO  - Processing Started for NVD CVE - 2010
2019-02-18 08:55:11,328 org.owasp.dependencycheck.data.update.nvd.ProcessTask:160
INFO  - Processing Complete for NVD CVE - 2002  (38305 ms)
2019-02-18 08:55:11,328 org.owasp.dependencycheck.data.update.nvd.ProcessTask:149
INFO  - Processing Started for NVD CVE - 2019
2019-02-18 08:55:15,470 org.owasp.dependencycheck.data.update.nvd.ProcessTask:160
INFO  - Processing Complete for NVD CVE - 2019  (4142 ms)
2019-02-18 08:55:15,471 org.owasp.dependencycheck.data.update.nvd.ProcessTask:149
INFO  - Processing Started for NVD CVE - 2015
2019-02-18 08:55:18,745 org.owasp.dependencycheck.data.update.nvd.ProcessTask:160
INFO  - Processing Complete for NVD CVE - 2009  (34145 ms)
2019-02-18 08:55:18,745 org.owasp.dependencycheck.data.update.nvd.ProcessTask:149
INFO  - Processing Started for NVD CVE - 2011
2019-02-18 08:55:19,051 org.owasp.dependencycheck.data.update.nvd.ProcessTask:160
INFO  - Processing Complete for NVD CVE - 2006  (45847 ms)
2019-02-18 08:55:19,051 org.owasp.dependencycheck.data.update.nvd.ProcessTask:149
INFO  - Processing Started for NVD CVE - 2017
2019-02-18 08:55:20,696 org.owasp.dependencycheck.data.update.nvd.ProcessTask:160
INFO  - Processing Complete for NVD CVE - 2008  (47504 ms)
2019-02-18 08:55:20,696 org.owasp.dependencycheck.data.update.nvd.ProcessTask:149
INFO  - Processing Started for NVD CVE - 2018
2019-02-18 08:55:42,014 org.owasp.dependencycheck.data.update.nvd.ProcessTask:160
INFO  - Processing Complete for NVD CVE - 2007  (50648 ms)
2019-02-18 08:55:42,014 org.owasp.dependencycheck.data.update.nvd.ProcessTask:149
INFO  - Processing Started for NVD CVE - Modified
2019-02-18 08:55:45,266 org.owasp.dependencycheck.data.update.nvd.ProcessTask:160
INFO  - Processing Complete for NVD CVE - Modified  (3251 ms)
2019-02-18 08:55:45,266 org.owasp.dependencycheck.data.update.nvd.ProcessTask:149
INFO  - Processing Started for NVD CVE - 2013
2019-02-18 08:55:55,132 org.owasp.dependencycheck.data.update.nvd.ProcessTask:160
INFO  - Processing Complete for NVD CVE - 2010  (50623 ms)
2019-02-18 08:55:55,132 org.owasp.dependencycheck.data.update.nvd.ProcessTask:149
INFO  - Processing Started for NVD CVE - 2016
2019-02-18 08:56:29,673 org.owasp.dependencycheck.data.update.nvd.ProcessTask:160
INFO  - Processing Complete for NVD CVE - 2011  (70927 ms)
2019-02-18 08:56:29,674 org.owasp.dependencycheck.data.update.nvd.ProcessTask:149
INFO  - Processing Started for NVD CVE - 2012
2019-02-18 08:56:37,773 org.owasp.dependencycheck.data.update.nvd.ProcessTask:160
INFO  - Processing Complete for NVD CVE - 2015  (82302 ms)
2019-02-18 08:56:37,774 org.owasp.dependencycheck.data.update.nvd.ProcessTask:149
INFO  - Processing Started for NVD CVE - 2014
2019-02-18 08:56:56,494 org.owasp.dependencycheck.data.update.nvd.ProcessTask:160
INFO  - Processing Complete for NVD CVE - 2013  (71228 ms)
2019-02-18 08:57:13,235 org.owasp.dependencycheck.data.update.nvd.ProcessTask:160
INFO  - Processing Complete for NVD CVE - 2016  (78103 ms)
2019-02-18 08:57:15,000 org.owasp.dependencycheck.data.update.nvd.ProcessTask:160
INFO  - Processing Complete for NVD CVE - 2017  (115949 ms)
2019-02-18 08:57:15,038 org.owasp.dependencycheck.data.update.nvd.ProcessTask:160
INFO  - Processing Complete for NVD CVE - 2012  (45364 ms)
2019-02-18 08:57:16,245 org.owasp.dependencycheck.data.update.nvd.ProcessTask:160
INFO  - Processing Complete for NVD CVE - 2018  (115549 ms)
2019-02-18 08:57:18,326 org.owasp.dependencycheck.data.update.nvd.ProcessTask:160
INFO  - Processing Complete for NVD CVE - 2014  (40552 ms)
2019-02-18 08:57:18,327 org.owasp.dependencycheck.data.update.NvdCveUpdater:303
INFO  - Begin database maintenance.
2019-02-18 08:57:41,624 org.owasp.dependencycheck.data.update.NvdCveUpdater:305
INFO  - End database maintenance.
2019-02-18 08:57:44,181 org.owasp.dependencycheck.Engine:914
INFO  - Check for updates complete (197388 ms)
2019-02-18 08:57:45,780 org.owasp.dependencycheck.Engine:1099
ERROR - Unable to continue dependency-check analysis.
2019-02-18 08:57:45,785 org.owasp.dependencycheck.App:198
ERROR - One or more fatal errors occurred
2019-02-18 08:57:45,785 org.owasp.dependencycheck.App:204
ERROR - No documents exist

**Comment by albuch** [comment]
See #1732 for a workaround.
Please note that the latest docker image is a milstone release (see https://github.com/jeremylong/DependencyCheck/releases). You might want to use v4.0.2 until 5.0.0 is released

**Comment by jamesratzlaff** [comment]
I was able to duplicate this issue when using a .properties file (based off of 4.0.3-SNAPSHOT) to explicitly redefine the defaults defined in dependencycheck.properties.  The setting that appears to cause it is by using
```
#defined in 4.0.3-SNAPSHOT dependencycheck.properties file
data.connection_string=jdbc:h2:file:%s;MV_STORE=FALSE;AUTOCOMMIT=ON;LOG=0;CACHE_SIZE=65536;
```
instead of
```
#defined in 5.0.0-M2 dependencycheck.properties file
data.connection_string=jdbc:h2:file:%s;AUTOCOMMIT=ON;LOG=0;CACHE_SIZE=65536;
```
It appears that having **MV_STORE=FALSE** in the connection String may be a cause of unpleasant things happening. So if anyone is having this issue be sure to double check any properties you may have merged with the defaults.

**Comment by jeremylong** [comment]
I believe this was fixed with the 5.0.0-M2 release.
